### DISCUSSION TITLE
changing the way {DISPLAYNAME} var works

### DISCUSSION THREAD

**Discussion by ghost** [EssentialsX/Essentials#4415] [Ideas and Feature Suggestions] [answered]
Currently the {displayname} variable in combination with LP. Does not come with a space between the prefix and username. 
This cannot be solved by simply adding a space to the prefix set in LP. Because this will interfere with having multiple prefixes (See screenshot) 

![image](https://user-images.githubusercontent.com/8274718/127722170-14a753d3-1406-4e90-a50a-c520b510b256.png)
--> Multiple prefixes and each prefix in LP got and extra space "[Rank] ". You see there is added one space to much.

![image](https://user-images.githubusercontent.com/8274718/127722059-d126462f-3cfa-4a42-808c-77e55f610d73.png) --> Main issue. All LP ranks did not get a space in the prefix configuration. But by using {prefix} {nickname} in chat the matter can be solved in chat. But not in the nickname message. Because I am not able to modify it in any way.

Hopefully you understand my problem and are maybe able to change the {displayname} var in a way so there is a space between them. Thanks!

**Comment by JRoy** [comment, marked as answer]
there isn't a way to fix this, we just show the displayname, luckperms is the one setting it without a space.
